pennylane/ops/__init__.py

```diff
@@ -21,6 +21,7 @@
 from .cv import *
 from .qubit import *
 from .channel import *
+from .arithmetic import *
```

Comment:
I switched to naming the submodule `op_math`. "arithemtic" could be confused with the operations like `QubitSum` and `QubitCarry`.